docs/notes/bugfix-16397.md

```diff
@@ -0,0 +1,18 @@
+# Change in empty is among the items of `1,2,3,` behavior
+
+In previous versions of LiveCode `is among the items of` behaved
+incorrectly when a trailing delimiter was present. Trailing delimiters
+are generally ignored in the LiveCode engine and legacy behaviors where
+they are not are documented anomalies that are maintained for backwards
+compatibility reasons. This change resolves the anomaly
+[bug report](http://quality.livecode.com/show_bug.cgi?id=16397)
```

Comment:
Can you put the bug report here in brackets or something? Otherwise it sounds like the bug report is anomalous.